runtime/tests/integration_test.rs

```diff
@@ -1128,3 +1128,36 @@ fn is_candidate_via_precompile() {
 			);
 		})
 }
+
+// #[test]
+// fn min_nomination_via_precompile() {
+// 	ExtBuilder::default()
+// 		.build()
+// 		.execute_with(|| {
+// 			let staking_precompile_address = H160::from_low_u64_be(2048);
+
+// 			let mut get_min_nom = Vec::<u8>::from([0u8; 4]);
+// 			get_min_nom[0..4].copy_from_slice(&hex_literal::hex!("c9f593b2"));
+
+// 			let min_nomination = 5u128;
+// 			let expected_min: U256 = min_nomination.into();
+// 			let mut buffer = [0u8; 32];
+// 			expected_min.to_big_endian(&mut buffer);
+// 			let expected_result = Some(Ok((ExitSucceed::Returned, buffer.to_vec(), 0)));
+
+// 			assert_eq!(
+// 				MoonbeamPrecompiles::<Runtime>::execute(
```

Comment:
```
---- min_nomination_via_precompile stdout ----
thread 'min_nomination_via_precompile' panicked at 'assertion failed: `(left == right)`
  left: `Some(Ok((Returned, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 69, 99, 145, 130, 68, 244, 0, 0], 0)))`,
 right: `Some(Ok((Returned, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5], 0)))`', runtime/tests/integration_test.rs:1149:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Looks like a decimal place issue. I guess the top one is how many wei are in 5 ether.

Reply:
That was it, thanks!